deploy/rbac-clowdapp.yml

```diff
@@ -78,13 +78,29 @@ objects:
           name: default-role-config
         - mountPath: /opt/rbac/rbac/management/role/permissions
           name: model-access-permissions
+        - mountPath: /opt/rbac/management/principal/umb_certs/
+          name: umb-cert
+        - mountPath: /opt/rbac/management/principal/umb_certs/
+          name: umb-key
         volumes:
         - configMap:
             name: ${CONFIG_MAP_NAME}
           name: default-role-config
         - configMap:
             name: ${MODEL_ACCESS_PERMISSIONS}
           name: model-access-permissions
+        - name: umb-cert
+          secret:
+            secretName: service-accounts
+            items:
+            - key: umb-cert
+              path: cert.pem
+        - name: umb-key
+          secret:
+            secretName: service-accounts
```

Comment:
I believe the value of `secretName` should be unique

Reply:
I think you are right, moved them into one